ci/shell.nix

```diff
@@ -0,0 +1,29 @@
+# nixos-unstable from 2020-06-02
```

Comment:
Do we need to be on unstable or do we want to?

Reply:
I guess it's just we want to. Newer pipenv (2020) is there, but otherwise not too much.